### ISSUE TITLE
[OCP 3.10] Add questions to all scenarios for OCP version

### ISSUE THREAD

**Issue by netzzer** [red-hat-storage/openshift-cic#34]
Need to add question about what OCP release will be deployed to get inventory file options correct.
New Question could be:
```
What version of OpenShift Container Platform are you deploying? (valid answers are 3.9 or 3.10)
```
Answer = 3.9 
```
------------------------------------------------------------
   CNS 3.9 - Inventory File Creator 
------------------------------------------------------------
'' 
The output is NOT A COMPLETE Inventory File.
Created Inventory file options should be copied and pasted into
the larger openshift-ansible inventory file for your deployment.
 ''
1. Storage for Applications + Registry 
2. Storage for Applications + Logging
3. Storage for Applications + Metrics 
4. Storage for Applications + Registry + Logging + Metrics
5. Storage for Applications Only
------------------------------------------------------------
Enter your choice [1-5] : 
```
Answer = 3.10
```
------------------------------------------------------------
   OCS 3.10 - Inventory File Creator
------------------------------------------------------------
'' 
The output is NOT A COMPLETE Inventory File.
Created Inventory file options should be copied and pasted into
the larger openshift-ansible inventory file for your deployment.
 ''
1. Storage for Applications + Registry 
2. Storage for Applications + Logging
3. Storage for Applications + Metrics 
4. Storage for Applications + Registry + Logging + Metrics
5. Storage for Applications Only
------------------------------------------------------------
Enter your choice [1-5] : 
```

**Comment by netzzer** [comment]
Resulting inventory file options for Answer = 3.10 correct option/value syntax can be found in Issues #18, #19, #20, #25, and #26.

**Comment by netzzer** [comment]
As a result of two cluster testing new OCP 3.10 requirement with details for fix in Issue #37.

**Comment by ramkrsna** [comment]
deploying (3.9 or 3.10)?
